cns/ipamclient/ipamclient_windows.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2017 Microsoft. All rights reserved.
+// MIT License
+
+// +build windows
+
+package ipamclient
+
+const (
+	defaultIpamPluginURL = "http://localhost:48080"
```

Comment:
It is not a good idea to hard code this. You should read it from the plugin spec file. This will also allow you to remove ipamclient_windows vs _linux versions. If the spec file does not exist, it is Linux.